gradle/toolchains.gradle

```diff
@@ -94,6 +94,25 @@ pluginManager.withPlugin("me.champeau.jmh") {
 	}
 }
 
+// Configure the JCStress plugin to use the toolchain for generating and running JCStress bytecode
+pluginManager.withPlugin("io.github.reyerizo.gradle.jcstress") {
+  if (testToolchainConfigured()) {
+	tasks.matching { it.name.contains('jcstress') && it.hasProperty('javaLauncher') }.configureEach {
+	  javaLauncher.set(javaToolchains.launcherFor {
+		languageVersion.set(testToolchainLanguageVersion())
+	  })evrika18
```

Comment:
`evrika18` looks like a syntax error, can you explain this?

Reply:
fixed